assets/src/application/ui/input/EspressoButton/EspressoButton.tsx

```diff
@@ -1,20 +1,20 @@
 import React from 'react';
-import classNames from 'classnames';
-import { Button } from 'antd';
 import AntIcon from '@ant-design/icons';
+import { Button } from 'antd';
+import classNames from 'classnames';
 
 import { EspressoButtonProps, EspressoButtonSize, EspressoButtonType, Icon } from './types';
-import { EspressoIcon, isEspressoIcon } from '../../display';
-import { isEmpty } from '@appServices/utilities/string';
-import Tooltip from './Tooltip';
+import { EspressoIcon, isEspressoIcon, withLabel, withTooltip } from '../../display';
+
+type ButtonType = React.ComponentType;
 
 /**
  * Button wrapper for adding styles
  *
  * forwardRef to be able to accept
  * onMouseEnter, onMouseLeave, onFocus, onClick events from parent
  */
-const EspressoButton = React.forwardRef<Button, EspressoButtonProps>(
+const EspressoButton = React.forwardRef<ButtonType, EspressoButtonProps>(
```

Comment:
This should be `Button` not `ButtonType`
```suggestion
const EspressoButton = React.forwardRef<Button, EspressoButtonProps>(
```